docs/root/version_history/current.rst

```diff
@@ -4,6 +4,7 @@
 Changes
 -------
 
+* api v2: added `envoy.api.enable_deprecated_v2_api_warning` runtime feature to enable warnings on api v2 use.  
```

Comment:
See how feature flags are phrased elsewhere. This should be phrased more like "we now log a warning for v2 use, this can be temporarily disabled via the following runtime feature flag", etc. Thanks you!

/wait